src/Tribe/Templates.php

```diff
@@ -280,18 +280,50 @@ public static function setup_ecp_template( $query ) {
 
 			if ( self::is_main_loop( $query ) && self::$wpHeadComplete ) {
 
+				/**
+				 * To solve all the Filters problems happening on `the_content` hook we saw the need to free the default
+				 * space used before from the new priority (5) and the old default (10) to prevent HTML bugs but also
+				 * allow other Content Managers to control the output on The Events Calendar pages
+				 */
+				self::priority_darkmagic_on_hook( 'the_content' );
+
 				// on loop start, unset the global post so that template tags don't work before the_content()
 				add_action( 'the_post', array( __CLASS__, 'spoof_the_post' ) );
 
 				// on the_content, load our events template
-				// We're hooking to priority 9 for better compatibility with other non-tribe plugins that hook to the_content
-				add_filter( 'the_content', array( __CLASS__, 'load_ecp_into_page_template' ), 9 );
+				// We're hooking to priority 5 for better compatibility with other non-tribe plugins that hook to the_content
+				add_filter( 'the_content', array( __CLASS__, 'load_ecp_into_page_template' ), 5 );
 
 				// remove the comments template
 				add_filter( 'comments_template', array( __CLASS__, 'load_ecp_comments_page_template' ) );
 
 				// only do this once
 				remove_action( 'loop_start', array( __CLASS__, 'setup_ecp_template' ) );
+
+			}
+		}
+
+		/**
+		 * Re prioritizes all hooks on a range of priorities for the given Hook
+		 *
+		 * CAUTION: Please only use this method if you really know what you are doing
+		 *
+		 * @return void
+		 */
+		private static function priority_darkmagic_on_hook( $hook_name, $start = 5, $end = 10 ) {
+			if ( ! empty( $GLOBALS['wp_filter'][ $hook_name ] ) ) {
+				$hooks = $GLOBALS['wp_filter'][ $hook_name ];
+			} else {
+				$hooks = array();
+			}
+
+			foreach ( $hooks as $priority => $hook ) {
+				if ( $priority <= $start || $priority > $end ) {
+					continue;
+				}
+				unset( $GLOBALS['wp_filter'][ $hook_name ][ $priority ] );
+
+				$GLOBALS['wp_filter'][ $hook_name ][ $priority - $start ] = $hooks[ $priority ];
```

Comment:
It feels like this oversteps some boundaries.  This seems like it could cause some real confusion and difficulties for other plugins/devs - especially when it comes to other plugins hooking to `the_content` after this method executes.

If priority `9` isn't working out, then maybe shifting back to `10` is the simple solution and accept some compatibility problems with some plugins. [C#37991](https://central.tri.be/issues/37991) is where the request to lower to 9 was made - one user vs a pile of others doesn't seem like a good trade-off for destructively adjusting hook priorities.